packages/css-library/src/stylesheets/elements.scss

```diff
@@ -5,3 +5,197 @@
 // p {
 //    @include @usa-paragraph
 // }
+
+@use "../../dist/tokens/scss/variables" as *;
+
+$color-link-default-hover: rgba($color-black, 0.05);
+
+/* 
+ Styles that are specific to VA.gov using CSS type selectors (elements/tags).
+*/
+
+.feature {
```

Comment:
This might not be a fully inclusive list but there are many places this class is being used right now:

<img width="704" alt="Screenshot 2023-12-11 at 2 53 30 PM" src="https://github.com/department-of-veterans-affairs/component-library/assets/872479/d82275e0-6a74-4d62-a52d-87a3378b3829">
<img width="710" alt="Screenshot 2023-12-11 at 2 53 15 PM" src="https://github.com/department-of-veterans-affairs/component-library/assets/872479/734e6519-3727-463d-b72a-12e107670b52">

Reply:
Or we: 
1. do an audit for all instances where a utility class such as `.feature` or `.va-button` can be replaced directly with a web component.
2. make follow on tickets as a result of that analysis
3. update all such instances
4. remove now deprecated code from Formation. 

Actually, that's probably what you were suggesting in the first place @jamigibbs haha. That's sounds way better than my original proposal above. :homer_disappear:

Reply:
Yes I think I was hoping to just remove it from this PR completely and then we'd try and remove it from Formation in another other ticket (by replacing it with a web component). If we do truly find that `.feature` needs to be a global CSS style, we can add it to the CSS Library then.